test/test-applications/integrations/TestApplication.Smoke/Program.cs

```diff
@@ -19,6 +19,9 @@
 using System.Diagnostics;
 using System.Diagnostics.Metrics;
 using System.Net.Http;
+#if !NETFRAMEWORK
```

Comment:
Can we remove these `#if !NETFRAMEWORK` in the test app? I think it should not ham. If it would then it would at least manifest a problem.

Reply:
This makes sense, removing `!NETFRAMEWORK`.